src/Button/ZoomButton/ZoomButton.spec.tsx

```diff
@@ -16,60 +20,71 @@ describe('<ZoomButton />', () => {
   });
 
   it('can be rendered', () => {
-    const wrapper = TestUtil.mountComponent(ZoomButton, {map});
-    expect(wrapper).not.toBeUndefined();
+    const { container } = render(<ZoomButton map={map} />);
+    expect(container).toBeVisible();
   });
 
   it('zooms in when clicked', () => {
-    const wrapper = TestUtil.mountComponent(ZoomButton, {map});
+    render(
+      <ZoomButton map={map}>Zoom test</ZoomButton>
+    );
 
     const initialZoom = map.getView().getZoom();
-
-    wrapper.instance().onClick();
+    const button = screen.getByText('Zoom test');
+    userEvent.click(button);
 
     const promise = new Promise(resolve => {
       setTimeout(resolve, 300);
     });
 
-    expect.assertions(1);
     return promise.then(() => {
       const newZoom = map.getView().getZoom();
       expect(newZoom).toBe(initialZoom + 1);
     });
   });
 
   it('can be configured to zoom out', () => {
-    const wrapper = TestUtil.mountComponent(ZoomButton, {map, delta: -1});
+    render(
+      <ZoomButton map={map} delta={-1}>Zoom test</ZoomButton>
+    );
 
     const initialZoom = map.getView().getZoom();
-
-    wrapper.instance().onClick();
+    const button = screen.getByText('Zoom test');
+    userEvent.click(button);
 
     const promise = new Promise(resolve => {
```

Comment:
Not related to the changes in particular, but we may want to make use of ours [actSetTimeout()](https://github.com/terrestris/react-geo/blob/master/src/Util/rtlTestUtils.tsx#L8) here.

Reply:
didn't know that existed, fixed